libs/auth-api-lib/src/lib/delegations/admin/delegation-admin-custom.service.ts

```diff
@@ -128,10 +260,36 @@ export class DelegationAdminCustomService {
         })
       }
 
+      await this.delegationDelegationTypeModel.destroy({
```

Comment:
Didn´t you use `ON CASCADE DELETE` for the `delegation_delegation_type`? That would take care of removing the row here.